app/src/main/java/com/vladglush/lab5/entity/UfcFighter.java

```diff
@@ -0,0 +1,53 @@
+package com.vladglush.lab5.entity;
+
+public class UfcFighter {
+    // variable names must be the same as from API responce
+
+    private final String first_name;
+    private final String last_name;
+    private final String thumbnail;
+    private final String weight_class;
```

Comment:
Look for Java code convention.

Reply:
https://google.github.io/gson/apidocs/com/google/gson/annotations/SerializedName.html